src/clang-c-frontend/clang_c_convert.h

```diff
@@ -72,6 +73,7 @@ class clang_c_convertert
   namespacet ns;
   std::vector<std::unique_ptr<clang::ASTUnit>> &ASTs;
   const messaget &msg;
+  const char *mode;
```

Comment:
Could you add a comment, e.g., "Default value for `symbolt::mode` specifying the definition's source language"? And if this member could be of type `irep_idt`, we'd save repeated lookups in the string_containert.